Makefile

```diff
@@ -120,13 +132,21 @@ prove : $(tests) $(examples)
 
 examples : $(examples)
 
-ceed.pc : ceed.pc.template
-	@sed 's:%prefix%:$(abspath .):' $< > $@
+$(ceed.pc) : pkgconfig-prefix = $(abspath .)
+$(OBJDIR)/ceed.pc : pkgconfig-prefix = $(prefix)
+.INTERMEDIATE: $(OBJDIR)/ceed.pc
+%/ceed.pc : ceed.pc.template | $$(@D)/.DIR
+	@sed "s:%prefix%:$(pkgconfig-prefix):" $< > $@
+
+install : $(libceed) $(OBJDIR)/ceed.pc
+	$(INSTALL_DATA) -D -t "$(DESTDIR)$(includedir)/" include/ceed.h 
```

Comment:
The BSD version of `install` on my Mac doesn't have these options (and fails). Isn't it simple to just copy these files?

Reply:
Permissions need to be set (in case umask is not 022 in the build environment) on the installed objects without race conditions. I pushed 1e25a746b1a31e71aaccbb0332aa28240dc5d54d which I have now tested on OSX.